homeassistant/components/comelit/alarm_control_panel.py

```diff
@@ -128,20 +129,38 @@ def alarm_state(self) -> AlarmControlPanelState | None:
             AlarmAreaState.TRIGGERED: AlarmControlPanelState.TRIGGERED,
         }.get(self._area.human_status)
 
+    async def _async_update_state(self, area_state: AlarmAreaState, armed: int) -> None:
+        """Update state after action."""
+        self._area.human_status = area_state
```

Comment:
this is setting the state directly, right? isn't the state updated from the device automatically after a bit?

Reply:
Yes but for 5 seconds you still see the old value on the UI thinking that the action didn't work

Reply:
Calling `self.coordinator.async_request_refresh()` instead, like the light and switch platforms do, does not make that quicker?

Reply:
BTW, I do something similar for climate and humidifier platforms

Reply:
I was wonder if we should still call `self.coordinator.async_request_refresh()` to ensure the states are fresh?